screens/MultibooksScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import { useLazyQuery, useQuery } from '@apollo/client';
+
+import Loader from '../components/Loader';
+
+import { SubjectQuery } from '../types/subject';
+import { formatSubjects } from '../utils/functions/sorts';
+import { BookQuery } from '../types/book';
+
+import GET_SUBJECTS from '../graphQL/subject';
+
+import { GET_BOOKS_BY_SUBJECT } from '../graphQL/books';
+import DropdownComponent from '../components/UI/DropDown';
+import MultiBooksRender from '../components/MultiBooksRender';
+import ErrorCustom from '../components/ErrorCustom';
+import findTargetLevels from '../utils/functions/findTargetLevels';
+
+export default function MultiBooksScreen() {
+  const [selectedSubject, setSelectedSubject] = React.useState<{
+    label: string;
+    value: string;
+  }>({ label: '', value: '' });
+
+  const { data, loading, error } = useQuery<SubjectQuery>(GET_SUBJECTS);
+
+  const [
+    GetBooksFromSubject,
```

Comment:
```suggestion
    getBooksFromSubject,
```
We don't capitalize function names, except it they are components :)

Reply:
@AlexButeau Even if they are query name ? 
In the doc i found that query names so i just believe that was a convention : 
https://www.apollographql.com/docs/react/get-started
const GET_LOCATIONS = gql`
  query GetLocations {
    locations {
      id
      name
      description
      photo
    }
  }
`;